app/views/publishers/jobseeker_profiles/search/_results.html.slim

```diff
@@ -21,3 +21,7 @@
           - summary_list.row do |row|
             - row.key text: t("publishers.jobseeker_profiles.preferred_working_patterns", count: preferences.working_patterns.count), classes: "govuk-body-s govuk-!-font-weight-bold govuk-!-padding-bottom-0"
             - row.value text: jobseeker_profile.job_preferences.all_working_patterns, classes: "govuk-body-s govuk-!-padding-bottom-0"
+
+          - summary_list.row do |row|
+            - row.key text: t("publishers.jobseeker_profiles.preferred_subjects", count: preferences.subjects.count), classes: "govuk-body-s govuk-!-font-weight-bold govuk-!-padding-bottom-0"
```

Comment:
Had no idea of being able to use `count`  for singular/plural wording :open_mouth: